roboflow/util/folderparser.py

```diff
@@ -197,12 +203,27 @@ def _loadAnnotations(folder, annotations):
                 if parsedType:
                     ann["parsed"] = parsed
                     ann["parsedType"] = parsedType
+        elif extension == ".jsonl":
+            ann["parsed"] = _read_jsonl(f"{folder}{ann['file']}")
+            ann["parsedType"] = "jsonl"
         elif extension == ".csv":
             ann["parsedType"] = "csv"
             ann["parsed"] = _parseAnnotationCSV(f"{folder}{ann['file']}")
     return annotations
 
 
+def _read_jsonl(path):
+    data = []
+    with open(path) as file:
+        for line in file:
+            try:
+                json_object = json.loads(line.strip())
+                data.append(json_object)
+            except json.JSONDecodeError:
+                print(f"Warning: Skipping invalid JSON line in {path}")
```

Comment:
Skip empty lines (last line for example), because it is not valid JSON and it is not a real issue.

Also, it is easy to include invalid line number.

```suggestion
        for linenum, line in enumerate(file, 1):
            if not line:
                continue
            try:
                json_object = json.loads(line.strip())
                data.append(json_object)
            except json.JSONDecodeError:
                print(f"Warning: Skipping invalid JSON line in {path}:{linenum}")
```